### PR TITLE
#53490 adds missing test for xmlrpc function in functions.php

### PR DESCRIPTION
**Pull request by pbearne**
adds missing test for xmlrpc function in functions.php

Trac ticket: https://core.trac.wordpress.org/ticket/53490

### CODE REVIEW

**Comment by audrasjb** [comment]
Hi, thanks for the pull request. I committed some changes to this PR to address some Coding Standards issues.

**Review by audrasjb — APPROVED** [review]

File: tests/phpunit/tests/functions/xmlrpc.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @group xmlrpc
+ * @ticket 53490
+ */
+class Tests_function_xmlrpc extends WP_UnitTestCase {
```
Comment (costdev) [inline tests/phpunit/tests/functions/xmlrpc.php]:
```suggestion
class Tests_Functions_XMLRPC extends WP_UnitTestCase {
```

**Review by costdev — CHANGES_REQUESTED** [review]
Just one change to the class name and I think these look good.

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! I have updated the test method descriptions to be a bit more detailed.

Merged in https://core.trac.wordpress.org/changeset/53326.

**Pull request closed without merging**